omod/src/main/java/org/openmrs/module/cohort/web/resource/CohortMemberResource.java

```diff
@@ -134,7 +134,7 @@ public CohortMember getByUniqueId(String uuid) {
 	
 	@Override
 	public void purge(CohortMember cohortMember, RequestContext context) throws ResponseException {
-		boolean purge = Boolean.getBoolean(context.getParameter("purge"));
+		boolean purge = Boolean.parseBoolean(context.getParameter("purge"));
```

Comment:
Fix should be much simpler: just unconditionally call `purgeCohortMember`. The framework already handles the `purge=true` bit.

Reply:
Oh, okay, let me fix that. Thanks @ibacher

Reply:
@ibacher I have made the changes [here](https://github.com/openmrs/openmrs-module-cohort/pull/32/commits/061451d41379b751f109444f660e33d53c67f95b), requesting a review if the fix is done correctly